backend/climateconnect_main/settings.py

```diff
@@ -247,11 +248,20 @@
 IDEA_MENTION_TEMPLATE_ID_DE = env('IDEA_MENTION_TEMPLATE_ID_DE')
 JOINED_IDEA_TEMPLATE = env('JOINED_IDEA_TEMPLATE')
 JOINED_IDEA_TEMPLATE_DE = env('JOINED_IDEA_TEMPLATE_DE')
+WEEKLY_RECOMMENDATIONS_EMAIL = env('WEEKLY_RECOMMENDATIONS_EMAIL')
+WEEKLY_RECOMMENDATIONS_EMAIL_DE = env('WEEKLY_RECOMMENDATIONS_EMAIL_DE')
 PROJECT_JOIN_REQUEST_TEMPLATE_ID = env('PROJECT_JOIN_REQUEST_TEMPLATE_ID')
 PROJECT_JOIN_REQUEST_TEMPLATE_ID_DE = env('PROJECT_JOIN_REQUEST_TEMPLATE_ID_DE')
 
+WEEKLY_RECOMMENDATIONS_EMAIL = env('WEEKLY_RECOMMENDATIONS_EMAIL')
```

Comment:
You have same variable name identified twice

Reply:
Also I would recommend that we use same email address for all automated emails. I would say let's use existing environment variable we have `CLIMATE_CONNECT_SUPPORT_EMAIL`